modules/utils/src/merkle.spec.ts

```diff
@@ -18,17 +18,29 @@ describe("generateMerkleTreeData", () => {
       });
   };
 
-  it("should work for a single transfer", () => {
+  // TODO: proof and verification on new tree
+  it.skip("should work for a single transfer", () => {
```

Comment:
And these should not be `.skip` or have other important parts of the test commented out. (There's a few cases of this here, I won't comment on each)